packages/react-app/src/components/AddressedCard.jsx

```diff
@@ -17,7 +17,7 @@ export default function AddressedCard({ badges }) {
       >
         {badges && badges.length > 0 ? (
           badges.map(badge => {
-            const src = 'https://remix-project.mypinata.cloud/ipfs/' + badge.decodedIpfsHash
+            const src = 'https://ipfs2.ethdevops.io/ipfs/' + badge.decodedIpfsHash
```

Comment:
could you rollback this change for the moment? our ipfs is not yet ready..

Reply:
Ok. I will do that